mlir/python/mlir/extras/dialects/transform/__init__.py

```diff
@@ -146,3 +148,7 @@ def test_match_ops_single(module: OpHandle):
 
     if dump_script:
         print(named_sequence_op)
+
+
+sequence = region_op(transform.SequenceOp.__base__, terminator=transform.YieldOp)
```

Comment:
Not related to this patch, but I don't think I could explain to anyone why we have a dialects tree in the extras namespace as well. For my own edification, could you explain why these things can't exist under the regular dialect/transform/ namespace? (I continue to get a code smell for just how... sprawling the transform dialect machinery is and wish it were more contained)

Reply:
I'm not super happy about this either but the prevailing notion is the goings-on around here are brittle and thus an explicit "buyer beware" namespace is needed.

Reply:
I don't think we should be doing that, personally. I think this should live in the regular dialects/transform/ tree. Put it in an experimental package under there if needed. I've dealt with enough breakages and churn related to transform dialect usage that I doubt anyone would be able to differentiate between "this part of it is buyer beware". I don't think it needs to sprawl outside of its dialect.

(ftr - I'm perfectly fine with transform dialect being developed and even having an experimental vibe to it, but I would like it contained to its location)

Reply:
paging @ftynse :)